packages/fxa-shared/metrics/amplitude-event.1.schema.json

```diff
@@ -135,15 +151,19 @@
           ]
         },
         "utm_content": {
+          "description": "Marketing campaign content identifier for the first flow in the session. This parameter lets you know what call-to-action brought in traffic, and is used to determine which call-to-action is performing better (i.e., if a particular text link or button is resulting in a higher click-through rate).",
           "$ref": "#/properties/user_properties/definitions/utmPattern"
         },
         "utm_medium": {
+          "description": "Marketing campaign medium for the first flow in the session. This parameter defines which medium the visitor used to find your website. It is used to track the effectiveness of a marketing medium, as it can be used to distinguish which medium is bringing the most traffic or the highest quality traffic (e.g., utm_medium=email).",
           "$ref": "#/properties/user_properties/definitions/utmPattern"
         },
         "utm_source": {
+          "description": "Marketing campaign source for the first flow in the session. This parameter provides specific information regarding which sites are referring your traffic.",
           "$ref": "#/properties/user_properties/definitions/utmPattern"
         },
         "utm_term": {
+          "description": "Marketing campaign search term for the first flow in the session. This parameter informs which terms were used in search ads that have led customers to your site. If you find there are specific terms that are sending more customers to your website, you can increase your efforts to try to rank your website higher for those terms organically or through paid search.",
```

Comment:
Are these from Google Analytics' website? Maybe cite a reference? There are also FxA-provided definitions of these at  https://mozilla.github.io/ecosystem-platform/relying-parties/reference/metrics-for-relying-parties#metrics-related-query-parameters.

Reply:
I had pilfered from `fxa-auth-server/docs/metrics-events.md` and another source. I have updated the descriptions to reflect a mix of the reference in Ecosystem Platform and Google Analytics, which I have now cited.